includes/reader-revenue/class-reader-revenue-emails.php

```diff
@@ -0,0 +1,259 @@
+<?php
+/**
+ * Reader-revenue related emails.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Reader-revenue related emails.
+ */
+class Reader_Revenue_Emails {
+	const POST_TYPE          = 'newspack_rr_email';
+	const POST_TYPE_META     = 'newspack_email_type';
+	const EMAIL_TYPE_RECEIPT = 'receipt';
+
+	/**
+	 * Initialize.
+	 *
+	 * @codeCoverageIgnore
+	 */
+	public static function init() {
+		add_action( 'init', [ __CLASS__, 'register_cpt' ] );
+		add_action( 'init', [ __CLASS__, 'register_meta' ] );
+		add_filter( 'newspack_newsletters_email_editor_cpts', [ __CLASS__, 'register_email_cpt_with_email_editor' ] );
+	}
+
+	/**
+	 * Register the custom post type.
+	 *
+	 * @param array $email_editor_cpts Post type which should be edited as emails.
+	 */
+	public static function register_email_cpt_with_email_editor( $email_editor_cpts ) {
+		$email_editor_cpts[] = self::POST_TYPE;
+		return $email_editor_cpts;
+	}
+
+	/**
+	 * Register the custom post type for emails.
+	 */
+	public static function register_cpt() {
+		if ( ! current_user_can( 'edit_others_posts' ) ) {
+			return;
+		}
+
+		$labels = [
```

Comment:
(Non-blocking suggestion)

Same suggestion as above for all labels: "Customer Email" instead of just "Email"

Reply:
Changed to "Reader Revenue Emails", but the user won't see that unless they navigate to the generic CPT view.